packages/chart/src/components/PieChart.tsx

```diff
@@ -20,6 +20,36 @@ const enterUpdateTransition = ({ startAngle, endAngle }: PieArcDatum<any>) => ({
   endAngle
 })
 
+// Would put this in helpers but it changes based on the structure of the data
+// - here the pie needs to omit 'name' but line and bar omit "Date"
+function cleanData(data) {
```

Comment:
You know the pattern here by now. If these are the same, move them up to CdcChart and pass the fn through.

Reply:
Yep